build.gradle

```diff
@@ -18,11 +18,12 @@ dependencies {
     implementation 'org.jetbrains:annotations:20.1.0'
     runtimeOnly 'com.h2database:h2'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.9.1'
     runtimeOnly 'com.mysql:mysql-connector-j'
     compileOnly 'org.projectlombok:lombok'
     annotationProcessor 'org.projectlombok:lombok'
 }
 
-tasks.named('test') {
+test {
```

Comment:
If I remember correctly you wanted to change it once ;) Do you really need it? It's not important and it will be better to focus on other things than resolve later conflicts in merge with such things.

Of course the new dependence is correct and needed :)

Reply:
![Junit](https://user-images.githubusercontent.com/113088417/232190025-fc4c0051-fac3-4fd9-b074-4cb8f7fb82d5.gif)

I'm not reinventing the wheel, I took it from the Jetbrains video about setting up Junit5 for Gradle in Intellij.

Reply:
You don't have to be rude :( I just wrote that it's not necessary to change that. The version we have works, so it's just another way which is correct...

Reply:
I understand, I didn't mean to come off as rude 😃  . As far as I'm concerned, we're still learning and something new is always appearing. We should update the things we use. To show you that I'm not being unfriendly, we'll do it your way 👍

Reply:
It's not "my way" - someone made that in our project :) Mayby you are correct and it's wrong. I don't know. As you wrote, we're still learning. It's good that you found that difference, but if we explore the topic, I wish to know why do we need to change it? How does this affect the code? Is it just a recording issue?